pkg/daemon/update.go

```diff
@@ -2122,19 +2127,21 @@ func (dn *CoreOSDaemon) experimentalUpdateLayeredConfig() error {
 			// I write this in the image build now, so config checker still works AND I don't get stuck on it
 			os.Remove(dn.currentConfigPath)
 
+			// Check and perform node drain if required
+			if err := dn.drainIfRequired(actions, diffFileSet, getOstreeFileDataReadFunc(booted.Osname, activeChecksum), getOstreeFileDataReadFunc(staged.Osname, staged.Checksum)); err != nil {
+				return err
+			}
+
 			// TODO(jkyros): This is a hack until we figure out file "owners"/layers/precedence
 			// When you add files to an image (like we with ignition-liveapply), they get stuffed into /usr/etc/, so they are
 			// essentially "default files"
 			// This defaults the files we added to /etc/ so we know the 3-way merge will update them with the container versions from machineconfig
-			_, err = runGetOut("rsync", "-avh", "/usr/etc/", "/etc/")
+			// @jkyros moved this after the drain because drain was having issues sometimes, he doesn't fully understand why yet
+			_, err = runGetCombined("rsync", "-avh", "/usr/etc/", "/etc/")
```

Comment:
One problem with this is likely to be SELinux labels (yes, our friend).  I think rsync doesn't do anything with that, so we'll end up with the defaults, which are likely not right for `/etc.`

Ohh, and we'll end up resetting the modification times to zero.  That *may* confuse things.